src/main/java/me/zeroeightsix/fiber/api/tree/HasValue.java

```diff
@@ -12,10 +12,5 @@
      */
     T getValue();
 
-    /**
-     * Returns the class of the type of the value being held
-     * @return the class of the type of the value
-     */
     Class<T> getType();
```

Comment:
Why remove this doc?

Reply:
Mistake, I removed this method then added it back